include/librealsense2/h/rs_sensor.h

```diff
@@ -93,6 +95,7 @@ typedef enum rs2_format
     RS2_FORMAT_Y411            , /**< 12-bit per-pixel. */
     RS2_FORMAT_Y16I            , /**< 12-bit per pixel interleaved. 12-bit left, 12-bit right. */
     RS2_FORMAT_M420            , /**< 24-bit for every pixel: y for each pixel, and u,v data for every four pixels - packed as 2 lines of y, 1 line of u,v */
+    RS2_FORMAT_OCCUP           , /**< 8-bit raw image, for occupancy mapping*/
```

Comment:
Why do we need a new format?
Can't the user ask for RAW8?
What will the user ask when on pointcloud?

Reply:
Let's talk about these